lib/rex/post/meterpreter/ui/console/command_dispatcher/kiwi.rb

```diff
@@ -303,37 +304,50 @@ def cmd_kerberos_ticket_use(*args)
   end
 
   #
-  # Dump all the wifi profiles/credentials
+  # Dump all the shared wifi profiles/credentials
   #
-  def cmd_wifi_list(*args)
-    results = client.kiwi.wifi_list
+  def cmd_wifi_list_shared(*args)
+    interfaces_dir = '%AllUsersProfile%\Microsoft\Wlansvc\Profiles\Interfaces'
```

Comment:
That's a good question. I might do a bit of testing to see if you do actually need to run as SYSTEM or a high priv user to get access to these files. I'll get back to you!